src/YesSql.Core/Session.cs

```diff
@@ -900,7 +908,15 @@ public async ValueTask DisposeAsync()
 
             _disposed = true;
 
-            await CommitOrRollbackTransactionAsync();
+            try
```

Comment:
We can create an inner function instead of writing code twice

Reply:
No, one is async the other on is not